app/config/config_dev.yml

```diff
@@ -29,5 +29,5 @@ monolog:
         chromephp:
             type:   chromephp
             level:  info
-#swiftmailer:
-#    delivery_address: me@example.comdoctrine:
```

Comment:
I was thinking just leave this line commented out of the config file as a reminder of where to put the info if we want it back.